examples/Actor/ActorClient/Program.cs

```diff
@@ -86,8 +87,8 @@ public static async Task Main(string[] args)
             var res = await nonRemotingProxy.InvokeMethodAsync<MyData>("GetData");
 
             Console.WriteLine("Registering the timer and reminder");
-            await proxy.RegisterTimer();
-            await proxy.RegisterReminder();
+            await proxy.RegisterTimer(null);
+            await proxy.RegisterReminder(null);
```

Comment:
nit: if this is a default-valued parameter you don't have to pass null here.

Reply:
The ActorProxy complained about the interface having a default value and without that, the top level doesn't appear to accept the default parameter. I'll look again as maybe I was just being wrong, but that's the behavior I observed.